tests/src/Functional/ContentPersonRenderTest.php

```diff
@@ -0,0 +1,280 @@
+
+    // Assert required fields are present.
+    $content_banner = $crawler->filter('.bcl-content-banner');
+    $this->assertCount(1, $content_banner);
+
+    $this->assertEquals(
+      'Stefan Mayer',
+      $content_banner->filter('.card-title')->text()
+    );
+    $this->assertEquals(
+      'DG Test',
+      $content_banner->filter('.card-body > div.my-4 > span:nth-child(1)')->text()
+    );
+    $this->assertEquals(
+      'Director',
+      $content_banner->filter('.card-body > div.my-4 > span:nth-child(2)')->text()
+    );
+    $this->assertEquals(
+      'Germany',
+      $content_banner->filter('.card-body > div.my-4 > span:nth-child(3)')->text()
+    );
+
+    // Assert optional fields are not present.
+    // Assert no image is present.
+    $this->assertCount(0, $content_banner->filter('img'));
+
+    // Assert short description is not present.
+    $this->assertCount(0, $content_banner->filter('p'));
+
+    // Assert no social media links are present.
+    $this->assertCount(0, $content_banner->filter('a.standalone'));
+
+    $content = $crawler->filter('div.col-12.col-lg-10.col-xl-9.col-xxl-8');
+
+    // Assert additional information is not present.
+    $this->assertStringNotContainsString(
+      'Additional information',
+      $content->text()
+    );
+
+    // Assert documents are not present.
+    $this->assertStringNotContainsString(
+      'Related documents',
+      $content->text()
+    );
+
+    // Test person with all fields.
+    $node_with_all_fields = $this->createExamplePersonWithAllFields();
+    $this->drupalGet($node_with_all_fields->toUrl());
+    $crawler = $client->getCrawler();
+
+    $content_banner = $crawler->filter('.bcl-content-banner');
+
+    $this->assertEquals(
+      'Stefan Mayer',
+      $content_banner->filter('.card-title')->text()
+    );
+    $this->assertEquals(
+      'DG Test',
+      $content_banner->filter('.card-body > div.my-4 > span:nth-child(1)')->text()
+    );
+    $this->assertEquals(
+      'Director',
+      $content_banner->filter('.card-body > div.my-4 > span:nth-child(2)')->text()
+    );
+    $this->assertEquals(
+      'Germany',
+      $content_banner->filter('.card-body > div.my-4 > span:nth-child(3)')->text()
+    );
+
+    // Assert content banner image.
+    $image = $content_banner->filter('img');
+    $this->assertCount(1, $image);
+    $this->assertCount(1, $image->filter('.card-img-top'));
+    $this->assertStringContainsString(
+      'image-test.png',
+      $image->attr('src')
+    );
+    $this->assertEquals('Starter Image test alt',
+      $image->attr('alt')
+    );
+
+    $this->assertEquals(
+      'This field is used to add a short biography of the person.',
+      $content_banner->filter('p')->text()
+    );
+
+    $this->assertEquals(
+      'Twitter profile',
+      $content_banner->filter('div.mb-2 > div.mb-3 > a.standalone')->text()
+    );
+
+    $content = $crawler->filter('div.col-12.col-lg-10.col-xl-9.col-xxl-8');
```

Comment:
Note: Here the very specific selector is ok because we use it for _positive_ assertions.
Well actually these classes are crazy, but they are added in our template, and coming from netlify, so it is legitimate that we want to test them here.